### ISSUE TITLE
All dem monsters!

### ISSUE THREAD

**Issue by ghost** [SqueeG/awesomeTome#10]
Any thoughts on what we're gonna do with all the monsters? I believe Frank made a (very valid and sensible) post with respect to needing to have the monsters designed first, before anything else happens. In my view, this is quite important, and quite a few things (if I had my druthers) should be changed about them. Here's a preliminary list:
- The undead, construct, plant and vermin types need rethinking and reworking
- CR = HD for everything
- The CR system should be put on cleaner benchmarks (without stuff like dragons randomly breaking it)
- Modify one-trick ponies to be less one-trick. These kinds of fights are boring.
- Try and find a way of letting people play monsters without making us sad.
- A monster design system that doesn't make us cry.
- Monster advancement that doesn't make us cry.
- Some kind of background/lore for monsters that doesn't make us cry (also see my setting issue).
- A statblock for those monsters that's easy to use and read.

I'd also like to add a few monsters to the whole thing, but that's just me.

**Comment by ExplosiveRunes** [comment]
This is one thing I think 4e got right conceptually in some ways, but majorly fucked the implementation on. Here's some ideas.
- _Types_ - The types in 3.5 are fucked, they don't even indicate the same thing. Some of them are based on how the creature is shaped, some are where the creature is from, and some indicate its metabolism. None of those fucking things is mutually exclusive.
- _CR = HD_ - Take it a step further, just have level. A monsters saves, hp, bab, etc are all based on its level and 'type' (or whatever). If a creature needs to have more or less hp than level would indicate, thats a special ability of that monster. If a monster is particularly dangerous for its level, it gets the [Elite +#] tag, and is never suitable to be a PC.

I started, but only ever completed the fundamentals of, a monster system for my d20 heartbreaker. In it, the major components of a monster were Origin, Form, Role, and Types
- _Origin_ - Literally a descriptor of where the creatures species originates. If I recall I had Natural (the prime), Magical (the prime, but with magic. Mythical Creatures, experiments. Astral creatures too), Elemental (Duh), Immortal (Outsiders), Aberrant (Far Planers).  These origins each impart a few special features, and a few class skills, but mostly serve as qualifiers for feats, classes, effects, etc. I had them arranged in a hierarchy that overrode those lower origins so you never had to worry about a template making a magical creature back into a natural one or some such.
- _Form_ - Just two; Beast and Person.  Beasts are pretty much things that are not tool users, and are generally a-ok to kill as adventurers without worrying about it, regardless of alignment. Yeah, the Brony Paladins get angry when you kill unicorns, but people for the most part don't give a damn. Persons are.... people. Killing them is homicide of some form, and you can generally expect a person to exist within a social contract of some kind, and for their relatives to come after you if you murder-hobo them.  Beasts get bonus HP per racial level, persons get bonus skill points each racial level and can take. Beasts can only take beast classes.
- _Role_ - Similar to 4e roles, except actually fucking explained. This is essentially the class chassis for monsters. HP per level, skill points, saves, and BAB are all determined by a creatures role and level. If I recall I had Artillery, Brute, Tank, Skirmisher, Sneak, and Leader.
- _Types_ - Types are the new subtype. Just a set of features, like immunities or special rules, that you want neatly packaged up because you're going to use it a lot. I even had some types that did nothing (Canine, Cephalopod, etc) that only served as tags to interact with effects and abilities. Construct, undead, vermin, etc became types.

Because so much of the deterministic nature of the system, the idea was that you could plug in all of the stuff into a simple generator and have it output a monster that you just had to assign special abilities to. So for example, if you wanted to make a Illithid knock off, you'd plug in those things (and ability scores) and the top of the stat block would looks something like...

Squidmans - Level 8 Leader
Medium Aberrant Person
Cephalopod

Which is pretty simple, and conveys a lot of relevant information very quickly.

**Comment by Lokathor** [comment]
As much as I would love these things, some of them seem possible and some of them don't. Cleaning up what a monster type does and then editing some monsters to have new subtypes to regroup them around but ultimately keep them mostly the same is doable. Tome of Necromancy did that, we could do some more of that.

Saying that CR = HD = Level and then replacing all the monsters from there, etc, etc, ...That's a bit beyond our mandate I think. I mean it'd be nice to do... but who has ever even started such a project? There's like 300 monsters. That's a whole lot of design work _just to replace monsters_, not even counting the fact that more systems have to be made up to make all the monster generation possible.

As much as I would love to see it done, it doesn't seem a realistic task.

**Comment by Tarkisflux** [comment]
I agree with Lokathor here. Fixing types and subtypes should just get done while we're transcribing or whatever, because Tome of Necro and whatnot. And the rest would be really nice to have, but... 

Look, different hit dice and numeric progressions based on role instead of type is a non-trivial change, even if it is one I'd love to see. So is the HD = CR = Level change that I've been talking to MS about forever (though we could just incorporate some of his and Cid's monsters that do that already, pending licensing permission). But it's also a lot more work than in other places and a potential bottleneck to finishing, and that's been the sort of thing we've shied away from so far.

So are you volunteering to get these rolling and potentially finish them? We have a lot of monster transcriptions to do, and if you wanted to provide guidelines to others kicking in monster transcriptions you could probably make some serious progress. And the monster side of things is one I'd really like to see better handled, so I support larger changes on this side than in other places. But you'd better be really serious about doing it before we start down this path, because stopping midway is almost the same as never starting on SRD transcription at all.

**Comment by Lokathor** [comment]
Yeah, because incidentally while I converted a large portion of the SRD into LaTeX already and we can use that as a good base, I stopped partway through cleaning up the tables for spells and didn't do a single monster at all. Endless pages of the same sort of thing over and over will probably kill your desire to work on the project faster than you think.

**Comment by ghost** [comment]
Sure, I can do a few. I'll probably LaTeX them up sometime soon and push them to the repo. I just need a bit of help in figuring out how we wanna lay them out.

**Comment by SqueeG** [comment]
I asked this previously about classes.... but would a web form be of any help?

You have a form with inputs for monster name, hp, etc... fill them in and it spits out latex formatted code.

**Comment by ghost** [comment]
SqueeG: We first have to decide what layout we want! I would prefer to have something that's more easy to read than the default monster formatting, to be honest, as this buries a lot of abilities and puts things in an order that I'm not entirely happy with. (Seemingly only) I like the 'new' statblock introduced in the later years of 3.5, and would be happy to use it, but I need to know what everyone else feels about this matter.

**Comment by SqueeG** [comment]
Are you referring to the one dubbed "The New Stat Block" in [this article](https://www.wizards.com/default.asp?x=dnd/dd/20060707a)?

Or do you mean the [MMIV format](https://www.wizards.com/default.asp?x=dnd/ex/20060704a)?

**Comment by ExplosiveRunes** [comment]
Lokathor, on the spells: There's a tex file in the source named spells.tex where I used a few macros to format the srd spells section. I _think_ all of the tables in it are displaying correctly (though they may need to be repositioned to appear on the correct page) **except** for the summoning tables which I never formatted.  Between the two sources, might we actually already have a complete spell chapter? Here's a compiled pdf of just the spells section [for reference](https://www.dropbox.com/s/4caoys6vnl5luz1/spells%20example.pdf)

**Comment by ghost** [comment]
SqueeG: Aren't they the same? I meant the MMIV format, but I think that this _is_ the new stat block.

**Comment by SqueeG** [comment]
@mrsinister13 : Basically the same, some extremely minor changes. But they're completely insignificant.
I admit I didn't really compare them, just saw that MMIV was suppose to be different so I gave their example link. Heh  >_>

**Comment by ghost** [comment]
@SqueeG: So, I guess we're going with that one? Something else? If I have that, I can get to work.

**Comment by SqueeG** [comment]
@mrsinister13 :  I don't have a problem with that. If there's a more logical way (in your opinion) to list everything out, then do a mock up of a few monsters!

I could whip up a tex generating form as long as I know what is suppose to go in and what is suppose to come out. :)

We could always do stat blocks like 4e. :P [Those are pretty straight forward](http://4.bp.blogspot.com/-diE9clD0CLk/UFLNG8EDJvI/AAAAAAAAATE/sOtjdkhmp8I/s1600/DC+Kobold+L1+Minion+Update.png)

**Comment by ghost** [comment]
I'd avoid those - they're a huge pain to do in LaTeX (believe me, I've tried). I'll push one to the repository as soon as I can.

**Comment by Lokathor** [comment]
Yes. Tables are bad juju in LaTeX. Doing one and making it come out right is okay. Making 300 monsters all have similar looks in their tables is asking for crazy. Text-block based formatting is best for monsters. By a long shot.

**Comment by SqueeG** [comment]
I've been working on a thing rwlevent to Monstuz! ... hoping to push it somqhere between tonight and wednesday. :p

**Comment by Tarkisflux** [comment]
It's not too late to still get it in on Wed on this coast....

**Comment by SqueeG** [comment]
Haha, sorry. I tried to give myself a reasonable timeline. I've just been absolutely exhausted the past few days after work.

**Comment by SqueeG** [comment]
Do you guys know how to add columns to sinister's monster table?

**Comment by Tarkisflux** [comment]
Yes. You need to update the table declaration to include the additional columns, by turning:

\begin{longtabu} to 0.7\linewidth{X[1]}

into 

\begin{longtabu} to 0.7\linewidth{X X}

and add another "X" to the section for each additional column you need. You'll probably want to turn the 0.7 into 0.9 or 1.0 or something as well, to give the table additional width for the new data. After that you can go through and use normal column breaks in the rest of the table rows (i.e. "& <thing for new row>") to add the stuff that you want.

If you want to take the descriptive stuff at the bottom and have it span all of the rows, you'll need to use the \multicolumn{'num_cols'}{'alignment'}{'contents'} command. And you'll need to do it for each row that is supposed to span multiple columns. Additional informations on that one (or other table stuff) can be found at  http://en.wikibooks.org/wiki/LaTeX/Tables#Rows_spanning_multiple_columns

**Comment by SqueeG** [comment]
Sweet thanks.

**Comment by Tarkisflux** [comment]
I've started working on a 1, 2, and 3 column monster preload using MS's setup as a base. Should be ready in a day or 5 (depending on whether I get it done before I leave town tomorrow or not).

**Comment by Tarkisflux** [comment]
3 column doesn't work, but 1 and 2 column monster tables have been templatized and committed. It should be really really easy to do up a monster table now, and they should all look the same. We may want to do some formatting or shuffling of fields on them as well, but you can see examples in the new tome-beastiary-doc.pdf.

The templates themselves still need some actual instruction text, because I didn't want to do it yet. They should be easy enough to figure out for now though.
